src/read_pool.rs

```diff
@@ -419,7 +419,7 @@ pub fn build_yatp_read_pool<E: Engine, R: FlowStatsReporter>(
     engine: E,
     resource_ctl: Option<Arc<ResourceController>>,
     cleanup_method: CleanupMethod,
-) -> ReadPool {
+) -> (ReadPool, String) {
```

Comment:
seem the name `String` only used in test? Do we really need it?

Reply:
+1. What about add a test method that accepts the name as input parameter.

Reply:
I made a wrap for common usage.

Reply:
NIT: I think another way to do expose the name is to put a name field only for test in the ReadPool, but the current implementation is ok to me.